src/components/Bullet.tsx

```diff
@@ -258,13 +265,13 @@ const Bullet = ({
         // calculate position of thought for different font sizes
         // Table column 1 needs more space between the bullet and thought for some reason
         marginLeft: (fontSize - 9) * 0.5 - 11 - extendClickWidth - (isTableCol1 ? fontSize / 4 : 0),
-        marginBottom: -extendClickHeight - 2,
+        marginBottom: extendClickHeight - 2,
         paddingTop: extendClickHeight,
         paddingLeft: extendClickWidth,
         paddingBottom: extendClickHeight + 2,
         position: 'absolute',
         verticalAlign: 'top',
-        width: 4, // make the bullet wide enough to be clicked, but not enough to encroach on the editable
+        width: 7, // make the bullet wide enough to be clicked, but not enough to encroach on the editable
```

Comment:
This width is a legacy of the previous layout engine that used static positioning. Now that the Bullet is absolutely positioned, changing the width will not affect the layout. There is no risk of it encroaching on the editable.

I'm thinking we set the width so that the bullet is flush with the editable. That is, `-marginLeft - paddingLeft`.